### ISSUE TITLE
[BUG] volume-plus icon not working on US-P

### ISSUE THREAD

**Issue by spike886** [joBr99/nspanel-lovelace-ui#334]
### PROBLEM DESCRIPTION
On the US version in landscape, on the media card, when you try to tap on the icon "volume-plus", it's handled by the "arrow-right" and moves to the next screen instead of increasing the volume

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Provide your apps.yaml:
```

nspanel-comedor:
  module: nspanel-lovelace-ui
  class: NsPanelLovelaceUIManager
  config:
    panelRecvTopic: "tele/nspanel/RESULT"
    panelSendTopic: "cmnd/nspanel/CustomSend"
    model: us-p
    updateMode: "auto-notify"
    sleepTimeout: 20
    sleepTracking: group.all_humans
    screenBrightness: 100
    #sleepBrightness: 10
    sleepBrightness:
      - time: "2:00:00"
        value: 0
      - time: "23:00:00"
        value: 1
      - time: "sunset + 1:00:00"
        value: 10
      - time: "sunrise"
        value: 100
    locale: "es_AR" # used for translations in translations.py and for localized date if babel python package is installed
    screensaver:
      key: screensaver
      entity: weather.manuela_pedraza
      theme:
          autoWeather: true
      defaultCard: '{{ "navigate.cardMedia_comedor" if is_state("media_player.mass_chromecastaudio3218", "playing") or is_state("media_player.chromecastaudio3218", "playing") else "navigate.cardGrid_general" }}'
    cards:
      - type: cardGrid
        title: Luces
        entities:
          - entity: light.luz_sillon
          - entity: light.luz_mesa
          - entity: light.luz_oficina
          - entity: light.luz_pasillo_1
          - entity: light.luz_pasillo_2
          - entity: light.luz_cocina
        key: general
      - type: cardGrid
        title: extras
        entities:
          - entity: light.luz_mesa_ambilight
          - entity: light.luz_sillon_ambilight
          - entity: cover.perciana_comedor
          - entity: navigate.cardQR_invitado
            name: WIFI
            icon: mdi:wifi
        key: extra
      - type: cardThermo
        title: Aire
        entity: climate.aire_acondicionado_comedor
      - type: cardMedia
        entity: media_player.mass_chromecastaudio3218
        title: Musica
        key: comedor
    hiddenCards:
      - type: cardQR
        title: WiFi
        key: invitado
        qrCode: "WIFI:S:********;T:WPA;P:*********;;"
        entities:
          - entity: iText.********
            name: Name
            icon: mdi:wifi
          - entity: iText.*********
            name: Password
            icon: mdi:key


```
- [x] Go to Settings>Add-ons>AppDaemon>Log and then, provide the output of the log during your issue/bug occurs:
```
2022-07-05 22:34:06.051138 INFO nspanel-comedor: Searching for the following page as defaultPage: navigate.cardGrid_general
2022-07-05 22:34:06.055361 INFO nspanel-comedor: Result for the following page as defaultPage: <luibackend.config.Card object at 0x7f6ff0116350>
2022-07-05 22:34:06.059512 INFO nspanel-comedor: Started rendering of page 0 with type cardGrid
2022-07-05 22:34:06.063872 INFO nspanel-comedor: Sending MQTT Message: pageType~cardGrid
2022-07-05 22:34:06.093958 INFO nspanel-comedor: Sending MQTT Message: entityUpd~Luces~1|1~light~light.luz_sillon~���~17299~Luz Sill��n~0~light~light.luz_mesa~���~17299~Luz mesa~0~light~light.luz_oficina~���~17299~Luz Oficina~0~light~light.luz_pasillo_1~���~17299~Luz Pasillo 1~0~light~light.luz_pasillo_2~���~17299~Luz Pasillo 2~0~light~light.luz_cocina~���~63142~Luz Cocina~1
2022-07-05 22:34:08.409848 INFO nspanel-comedor: MQTT callback for: {'topic': 'tele/nspanel/RESULT', 'wildcard': None, 'payload': '{"CustomRecv":"event,buttonPress2,cardGrid,bPrev"}'}
2022-07-05 22:34:08.414191 INFO nspanel-comedor: Received Message from Screen: event,buttonPress2,cardGrid,bPrev
2022-07-05 22:34:08.417928 INFO nspanel-comedor: Button Press Event; entity_id: cardGrid; button_type: bPrev; value: None 
2022-07-05 22:34:08.421636 INFO nspanel-comedor: Started rendering of page 3 with type cardMedia
2022-07-05 22:34:08.425444 INFO nspanel-comedor: Sending MQTT Message: pageType~cardMedia
2022-07-05 22:34:08.437128 INFO nspanel-comedor: Sending MQTT Message: entityUpd~Musica~1|1~media_player.mass_chromecastaudio3218~���~~~0~���~~~1374~
2022-07-05 22:34:28.615142 INFO nspanel-comedor: MQTT callback for: {'topic': 'tele/nspanel/RESULT', 'wildcard': None, 'payload': '{"CustomRecv":"event,sleepReached,cardMedia"}'}
2022-07-05 22:34:28.619463 INFO nspanel-comedor: Received Message from Screen: event,sleepReached,cardMedia
2022-07-05 22:34:28.626108 INFO nspanel-comedor: Button Press Event; entity_id: cardMedia; button_type: sleepReached; value: None 
2022-07-05 22:34:28.631800 INFO nspanel-comedor: Started rendering of page None with type screensaver
2022-07-05 22:34:28.637402 INFO nspanel-comedor: Sending MQTT Message: pageType~screensaver
2022-07-05 22:34:28.670033 INFO nspanel-comedor: Sending MQTT Message: weatherUpdate~���~10.8��C~mi��~���~15.5��C~jue~���~13.6��C~vie~���~11.2��C~s��b~���~16.6��C~~
2022-07-05 22:34:28.676460 INFO nspanel-comedor: Sending MQTT Message: color~0~65535~65535~65535~35957~65535~65535~65535~65535~65535~31728~35957~33759~33759~65535~65535~65535~65535~65535~65535~65535~65535
2022-07-05 22:34:28.683076 INFO nspanel-comedor: Sending MQTT Message: X
2022-07-05 22:35:01.090506 INFO nspanel-comedor: Sending MQTT Message: time~22:35~
2022-07-05 22:36:01.022405 INFO nspanel-comedor: Sending MQTT Message: time~22:36~
2022-07-05 22:37:01.023121 INFO nspanel-comedor: Sending MQTT Message: time~22:37~
```

### TO REPRODUCE

1. Go to the media card
2. tab on the "volume-plus" icon

### EXPECTED BEHAVIOUR
The volume-plus icon show visual feedback showing that it is being tab and the volume should be increased

### ACTUAL BEHAVIOR
When you tab on the volume-plus icon, a visual feedback is shown on the arrow-right icon and the next page is shown

### SCREENSHOTS
![image](https://user-images.githubusercontent.com/5800938/177449635-c2a5852e-4d27-4518-895a-e7a3278b8579.png)

### ADDITIONAL CONTEXT
It can be reproducible most of the time, but sometimes when you tab, nothing happends

### PANEL / FIRMWARE VERSION
v3.1.1 US-P

**Comment by joBr99** [comment]
This is caused by the invisible button at the edge for the "edge swipe" feature.

I've changed the layout of cardMedia in the US-P Version, so this should be fixed.

![image](https://user-images.githubusercontent.com/29555657/177614960-1e949827-c165-4026-bb81-fe87d9cfb7c0.png)

**Comment by joBr99** [comment]
This will be available in 3.2.0, for now you have to go with the dev version if you want to test it.

https://docs.nspanel.pky.eu/faq/?h=devel#how-to-upgrade-from-a-release-to-the-current-development-version
